environments/openshift/manifests/observatorium-template.yaml

```diff
@@ -72,6 +72,92 @@ objects:
               cpu: ${THANOS_COMPACTOR_CPU_REQUEST}
               memory: ${THANOS_COMPACTOR_MEMORY_REQUEST}
     volumeClaimTemplates: []
+- apiVersion: v1
+  data:
+    observatorium-cache-config.yaml: |-
+      "auth_enabled": false
+      "frontend":
+        "align_queries_with_step": true
+        "cache_results": true
+        "compress_responses": true
+        "results_cache":
+          "cache":
+            "enable_fifocache": true
+            "fifocache":
+              "size": 1024
+              "validity": "6h"
+          "max_freshness": "1m"
+        "split_queries_by_day": true
```

Comment:
Be aware of this (!)  This will change performance characteristics of all components a lot.

Reply:
Thanks for the heads up. Since we have a number of querier pods in the backend for this cache/proxy, I'm hoping that this is more evenly distributing the queries? What other implications do you think we'll have?